openlibrary/templates/books/edit/edition.html

```diff
@@ -73,7 +73,9 @@
             $else:
                 <span class="edition_count">$item.edition_count editions</span>
         </div>
-
+$ translatable_strings = ['Please select a role.', 'You need to give this ROLE a name.', 'Please select an identifier.', 'You need to give a value to ID.', 'ID ids cannot contain whitespace.', 'ID must be exactly 10 characters [0-9] or X.', 'ID must be exactly 13 digits [0-9]. For example: 978-1-56619-909-4', 'Please select a classification.', 'You need to give a value to CLASS.']
+$ config = {translatable: _(translatable) for translatable in translatable_strings}
```

Comment:
This will unfortunately not work; the `_` function is computed statically, so it won't work with dynamic values. You'll need to build the `{}` directly. These strings won't be extracted correctly from the codebase.
```suggestion
$ config = {translatable: _(translatable) for translatable in translatable_strings}
```